modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/download/task/DownloadEpisodeTask.kt

```diff
@@ -64,7 +68,9 @@ class DownloadEpisodeTask @AssistedInject constructor(
     @Assisted params: WorkerParameters,
     var downloadManager: DownloadManager,
     var episodeManager: EpisodeManager,
-    var userEpisodeManager: UserEpisodeManager
+    var userEpisodeManager: UserEpisodeManager,
+    @DownloadCallFactory private val callFactory: Call.Factory,
+    @DownloadRequestBuilder private val requestBuilderProvider: Provider<Request.Builder>
```

Comment:
Depending on which app is running (phone/automotive/wear), these dependencies will be provided by the `@DownloadCallFactory` and `@DownloadRequestBuilder` provide methods from the relevant app (i.e., from `AppModule`, `AutomotiveAppModule` or `WearNetworkModule`).